### DISCUSSION TITLE
Authorization Error on localhost

### DISCUSSION THREAD

**Discussion by 87prashant** [ParabolInc/parabol#6776] [Q&A] [answered]
Not able to login to the account on localhost with google.

![Screenshot 2022-06-17 012313](https://user-images.githubusercontent.com/106697681/174154388-9f3431f5-6cb2-43b8-a992-38b8e4c7a908.png)

**Comment by tianrunhe** [comment, marked as answer]
Hi, @87prashant, to enable login with Google on your localhost:

1. Create your own Google authorization credentials (you can refer to [this guide](https://developers.google.com/identity/sign-in/web/sign-in) from Google)
2. Copy your `Client Id` and `Client Secret` in the console
3. Paste those values into your `.env`:
```
GOOGLE_OAUTH_CLIENT_ID='<Client Id>'
GOOGLE_OAUTH_CLIENT_SECRET='<Client Secret>'
```

**Reply by 87prashant** [reply]
Thanks @tianrunhe for help.

Can you help on this one?

> I am having the below errors, is this normal? 
> 
> I just followed the installation guide on readme.md
> `$ git clone https://github.com/ParabolInc/parabol.git`
> `$ cd parabol`
> `$ cp .env.example .env # Add your own vars here`
> `$ yarn`
> `$ yarn db:start`
> `$ yarn dev `
> 
> or am I missing something?
> 
> ![Screenshot 2022-06-17 005641](https://user-images.githubusercontent.com/106697681/174150368-3be2ee2a-cdf0-40d9-8743-887b129de34e.png)
